share/completions/rustup.fish

```diff
@@ -224,9 +224,8 @@ set -l __rustup_toolchains (rustup toolchain list | string replace -r "\s+.*" ''
 set -l __rustup_toolchains_short (__rustup_strip_common_suffix_strict $__rustup_toolchains)
 
 # Needs to be global to retain access from functions
```

Comment:
This comment would of course also need to be changed - "needs to be accessible to functions, hence the x" or something?

Reply:
Removed it since it should be self-explanatory